build.zig

```diff
@@ -138,6 +153,82 @@ pub fn build(b: *Build) !void {
                 lib_support_c_sources ++
                 lib_dxilcompression_c_sources;
 
+            // Build and link SPIRV-Tools
```

Comment:
definitely think we should be using https://github.com/hexops/spirv-tools here

Reply:
I'll definitely be using it instead.